tests/mmm/test_delayed_saturated_mmm.py

```diff
@@ -106,15 +106,15 @@ def mmm_with_fourier_features() -> DelayedSaturatedMMM:
     )
 
 
-@pytest.fixture(scope="class")
+@pytest.fixture(scope="module")
 def mmm_fitted(
     mmm: DelayedSaturatedMMM, toy_X: pd.DataFrame, toy_y: pd.Series
 ) -> DelayedSaturatedMMM:
     mmm.fit(X=toy_X, y=toy_y, target_accept=0.8, draws=3, chains=2)
```

Comment:
Instead of this nearly useless fit, you can take draws from the prior predictive and store those as if they were the posterior. We do this in almost all CLV tests.

There should be one test that actually calls fit and asserts it is converging to something correct, that can be marked with @pytest.mark.slow so it is skipped by default locally.

Reply:
I dunno. We can simulate data with known parameters and check the posterior converges to those with some relative tolerance.

Reply:
Cool. I will add that in

Some of the tests still call fit with draws=100
Is that an issue?

Reply:
Depends on whether that is relevant for the test or not